project_testing/housing_characteristics/Geometry Building Type FPL.tsv

```diff
@@ -1,2 +1,2 @@
 Option=Mobile Home	Option=Single-Family Detached	Option=Single-Family Attached	Option=2 Unit	Option=3 or 4 Unit	Option=5 to 9 Unit	Option=10 to 19 Unit	Option=20 to 49 Unit	Option=50 or more Unit
-0.111111111	0.111111111	0.111111111	0.111111111	0.111111111	0.111111111	0.111111111	0.111111111	0.111111111
+0.166666667	0.166666667	0.333333333	0.055555556	0.055555556	0.055555556	0.055555556	0.055555556	0.055555556
```

Comment:
What's the reason for this change?

Reply:
I was intending to sample one third SFD, one third SFA, one third MF. But with the fractions before, it was oversampling MF.

Reply:
Got it. 👍